src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -5,11 +5,24 @@ import { MessagePackHubProtocol } from '@aspnet/signalr-protocol-msgpack';
 import { OutOfProcessRenderBatch } from './Rendering/RenderBatch/OutOfProcessRenderBatch';
 import { internalFunctions as uriHelperFunctions } from './Services/UriHelper';
 import { renderBatch } from './Rendering/Renderer';
-import { fetchBootConfigAsync, loadEmbeddedResourcesAsync } from './BootCommon';
+import { fetchBootConfigAsync, loadEmbeddedResourcesAsync, shouldAutoStart } from './BootCommon';
 import { CircuitHandler } from './Platform/Circuits/CircuitHandler';
 import { AutoReconnectCircuitHandler } from './Platform/Circuits/AutoReconnectCircuitHandler';
 
-async function boot() {
+type SignalRBuilder = (builder: signalR.HubConnectionBuilder) => void;
+type BlazorOptions = {
+  configureSignalR: SignalRBuilder | null,
+};
+
+let started = false;
+
+async function boot(options?: BlazorOptions | null) {
```

Comment:
`| null` should not be needed

Reply:
Should we mark boot as export?

Reply:
out of scope for this PR. I'm not trying to solve the scenario of using our scripts as a module. Steve created an issue for it.